infra/macos/scripts/rebuild-crontask.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
This file is meant to run within the macOS CI nodes as part of their initialization. I believe we really do want to run `/bin//bash` here. Either way, touching `infra/macos` should generally be done in separate PRs because only @nycnewman can reasonably approve those.

Reply:
Thanks, I've reverted the change here.